Dockerfile

```diff
@@ -11,18 +11,21 @@ RUN dpkg --add-architecture i386 \
         lib32gcc1 \
         libsdl2-2.0-0 \
         libsdl2-2.0-0:i386 \
+        libc6-dev \
         ca-certificates \
         supervisor \
         procps \
         locales \
         unzip \
         zip \
         rsync \
+        wget \
```

Comment:
When you rebase on the current master you get wget from busybox. Added it for syslog and crond.